sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -277,26 +277,29 @@ private Combine() {
    *   public Accum createAccumulator() {
    *     return new Accum();
    *   }
+   *
    *   public Accum addInput(Accum accum, Integer input) {
    *       accum.sum += input;
    *       accum.count++;
    *       return accum;
    *   }
-   *   public Accum mergeAccumulators(Iterable<Accum> accums) {
+   *
+   *   public Accum{@literal mergeAccumulators(Iterable<Accum> accums)} {
```

Comment:
wrong I guess ?

Reply:
In this particular case, we're using `<pre><code>` for the overall block, which doesn't handle `<>`. The `@literal` added here is to accomodate the brackets in `<Accum>`.

Reply:
Fair enough as it is rendered well